src/api.js

```diff
@@ -3,8 +3,11 @@ const reviewsApi = axios.create({
   baseURL: "https://magnificent-hospital-gown-duck.cyclic.app/api",
 });
 
-export const getReviews = () => {
+export const getReviews = (category) => {
   let path = "/reviews";
+  if (category){
+    path+= `/?category=${category}`
+  }
   return reviewsApi.get(path).then((data) => {
```

Comment:
For a neat refactor, look up how to use axios params

Reply:
thanks, have done that much easier, i did try it but got confused the first time but understand it now